src/main/bash/generate-csv-fragments-images.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+version="$1"
+curl -f -o versions.yaml https://raw.githubusercontent.com/entando-k8s/entando-k8s-controller-coordinator/v${version}/charts/entando-k8s-controller-coordinator/templates/docker-image-info-configmap.yaml
+retcode=$?
+if [ $retcode -ne 0 ]; then
```

Comment:
@ampie same as above, are you really looking just for network errors?

Reply:
See above answer